pages/test_analytics/importing_json.md

```diff
@@ -215,32 +215,63 @@ end
   "result": "failed",
   "failure_reason": "Failure/Error: expect(true).to eq false",
   "failure_expanded": [
-  {
-      "expanded": [
-      "  expected: false",
-      "       got: true",
-      "",
-      "  (compared using ==)",
-      "",
-      "  Diff:",
-      "  @@ -1 +1 @@",
-      "  -false","  +true"
-      ],
-      "backtrace": [
-      "./spec/models/analytics/upload_spec.rb:25:in `block (3 levels) in \u003ctop (required)\u003e'","./spec/support/log.rb:17:in `run'",
-      "./spec/support/log.rb:66:in `block (2 levels) in \u003ctop (required)\u003e'",
-      "./spec/support/database.rb:19:in `block (2 levels) in \u003ctop (required)\u003e'",
-      "/Users/abc/Documents/rspec-buildkite-analytics/lib/rspec/buildkite/analytics/uploader.rb:153:in `block (2 levels) in configure'",
-      "-e:1:in `\u003cmain\u003e'"
-      ]
-  }
+    /* failure_expanded object */
   ],
   "history": {
     /* history object */
   }
 }
 ```
 
+### Failure expanded objects
+
+A failure expanded array contains extra details about the failed test.
+
+<table class="responsive-table">
+  <thead>
+    <tr>
+      <th>Key</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% TEST_ANALYTICS_JSON_FIELDS_FAILURE_EXPANDED['fields'].each do |field| -%>
+      <tr>
+        <td><code><%= field['name'] %></code> <%= '(required)' if field['required'] %></td>
+        <td><%= field['type'] %> <%= render_enumerated_values(field['enumerated_values']) %></td>
+        <td>
+          <%= render_markdown(text: field['desc']) %>
+        </td>
+      </tr>
+    <% end -%>
+  </tbody>
+</table>
+
+**Example:**
+
+```js
+{
+  "expanded": [
+  "  expected: false",
+  "       got: true",
+  "",
+  "  (compared using ==)",
+  "",
+  "  Diff:",
+  "  @@ -1 +1 @@",
+  "  -false","  +true"
+  ],
+  "backtrace": [
+    "./spec/models/analytics/upload_spec.rb:25:in `block (3 levels) in \u003ctop (required)\u003e'","./spec/support/log.rb:17:in `run'",
+    "./spec/support/log.rb:66:in `block (2 levels) in \u003ctop (required)\u003e'",
+    "./spec/support/database.rb:19:in `block (2 levels) in \u003ctop (required)\u003e'",
+    "/Users/abc/Documents/rspec-buildkite-analytics/lib/rspec/buildkite/analytics/uploader.rb:153:in `block (2 levels) in configure'",
+    "-e:1:in `\u003cmain\u003e'"
```

Comment:
Aside: @dannymidnight, the `\` in `\u003c` looks a bit strange when it renders:

![Screenshot 2023-08-21 at 5 14 00 pm](https://github.com/buildkite/docs/assets/119824349/136e30df-e2b7-47ae-95b9-b7f44db9f567)

Is that a bug?